src/test/java/gov/usgs/wma/waterdata/collections/CollectionParamsTest.java

```diff
@@ -0,0 +1,65 @@
+package gov.usgs.wma.waterdata.collections;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static gov.usgs.wma.waterdata.collections.CollectionParams.*;
+
+import java.util.Map;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import gov.usgs.wma.waterdata.ConfigurationService;
+
+class CollectionParamsTest {
+
+	private static final String TEST_SERVER_URL="/sever/url";
+
+	private String expectedCollectionId = "aCollectionId";
+	private String expectedFeatureId = "aFeatureId";
+	private String expectedTimeSeriesId = "aGUID";
+
+	private ConfigurationService config;
+	private CollectionParams builder;
+	private Map<String,Object> params;
+
+	@BeforeEach
+	public void before() {
+		config = new ConfigurationService();
+		config.setServerUrl(TEST_SERVER_URL);
+
+		builder = new CollectionParams(config);
+	}
+
+	private void commonAsserts() {
+		assertNotNull(params);
+		assertEquals(TEST_SERVER_URL, params.get(PARAM_SERVER_URL));
```

Comment:
This test can use config.getServerUrl() for the expected value. The code is expected to use that value.

Reply:
Thanks for adding the test.

Reply:
We certainly can, in an integration test. See prior comment.
Glad to add testing. I did not achieve full coverage, I covered essentially only in the area of what I was working.

Reply:
It has been agreed that this will be done when there is an integration test.